proto/minder/v1/minder.proto

```diff
@@ -1282,17 +1282,22 @@ message RuleType {
                     string action = 2;
                     // the content of the file
                     string content = 4;
-                    // the GIT mode of the file. Not UNIX mode! String because the GH API also uses strings
-                    // the usual modes are: 100644 for regular files, 100755 for executable files and
-                    // 040000 for submodules (which we don't use but now you know the meaning of the 1 in 100644)
+                    // The UNIX file permissions to set on the file
                     optional string mode = 3;
                 }
 
                 // the title of the PR
                 string title = 1;
                 // the body of the PR
                 string body = 2;
-                repeated Content contents = 3;
+                repeated Content contents = 4;
+                // the method to use to create the PR. For now, these are supported:
+                // -- minder.content - ensures that the content of the file is exactly as specified
+                //                     refer to the Content message for more details
+                // -- minder.frizbee.actions.replace_tags_with_sha - finds any github actions within a workflow
```

Comment:
I wonder if frizbee should be in the name or just `minder.actions.replace_tags_with_sha` to not expose that we use frizbee under the hood as an implementation detail.

Reply:
done, marking as resolved